### PR TITLE
Add Meta Llama 3.1, 3.2 instruct model enums to Bedrock

### PR DESCRIPTION
**Pull request by dafriz**
Add Meta Llama 3.1, 3.2 instruct model enums to Bedrock

Reference for model ids - https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html

### CODE REVIEW

File: models/spring-ai-bedrock/src/main/java/org/springframework/ai/bedrock/llama/api/LlamaChatBedrockApi.java [hunk 1]
```diff
@@ -226,7 +226,42 @@ public enum LlamaChatModel implements ChatModelDescription {
...
+
+		/**
+		 * meta.llama3-1-405b-instruct-v1:0
+		 */
+		LLAMA3_1_405B_INSTRUCT_V1("meta.llama3-1-405b-instruct-v1:0"),
+
+		/**
+		 * meta.llama3-2-1b-instruct-v1:0
+		 */
+		LLAMA3_2_1B_INSTRUCT_V1("meta.llama3-2-1b-instruct-v1:0"),
```
Comment (markpollack) [inline models/spring-ai-bedrock/src/main/java/org/springframework/ai/bedrock/llama/api/LlamaChatBedrockApi.java:249]: it's hard to find these strings from the docs.  I went [here](https://github.com/meta-llama/llama-stack/blob/8d41e6caa90c0e7bb44ec1755302a943bdf654f1/llama_stack/providers/adapters/inference/bedrock/bedrock.py#L23) to find the previously listed `llama3-1-x` models but dont' see the `llama-2-x` model values in the same code base.

Comment (markpollack) [reply models/spring-ai-bedrock/src/main/java/org/springframework/ai/bedrock/llama/api/LlamaChatBedrockApi.java:249]: There is also cross-region ids, see - https://docs.aws.amazon.com/bedrock/latest/userguide/cross-region-inference-support.html

Comment (markpollack) [reply models/spring-ai-bedrock/src/main/java/org/springframework/ai/bedrock/llama/api/LlamaChatBedrockApi.java:249]: ah, found the model listing here - https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html  added to the javadoc for future reference.

**Comment by markpollack** [comment]
thanks, merged in 250d2afbb63627359a7a1fb2c4d9847db7ac1e26

**Comment by markpollack** [comment]
merged in 250d2afbb63627359a7a1fb2c4d9847db7ac1e26

File: models/spring-ai-bedrock/src/main/java/org/springframework/ai/bedrock/llama/api/LlamaChatBedrockApi.java (hunk 1 above)
Comment (dafriz) [reply models/spring-ai-bedrock/src/main/java/org/springframework/ai/bedrock/llama/api/LlamaChatBedrockApi.java:249]: I see there is an issue raised about support for cross-region inference - https://github.com/spring-projects/spring-ai/issues/1282 Let's discuss there as this issue is closed.

**Pull request closed without merging**